### DISCUSSION TITLE
Need help with getting from a model

### DISCUSSION THREAD

**Discussion by SticksDev** [prisma/prisma#14240] [Q&A]
Hi, I'm quite new to prisma, and I'm trying to get something from a model, like so:

```
// Create a new prisma client
    const prisma = new PrismaClient();
    
    // Find the users's acess_token by their user ID (providerAccountId)
    const accessToken = await prisma.Account.findOne({
        where: {
            id: userID,
        },
        select: {
            accessToken: true,
            token_type: true,
        },
    })
```

But it spits out `TypeError: prisma.Account.findOne is not a function` I tried lowercase to no avail. Ideas?

**Comment by Yankue** [comment]
Since v2.12.0, `model.findOne` has been replaced with `model.findUnique`, so you just simply switch it to `prisma.Account.findUnique`.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
